README.md

```diff
@@ -23,16 +23,51 @@ To handle intentions, HAProxy Connect, sets up a SPOE filter on the application
 ## How to use
 
 ```
-haproxy-connect                                                                                      \
-    -sidecar-for <your-service-id>                                                                   \
-                                                                                                     \
-    -enable-intentions                      # wether or not to enbale intentions                     \
-
-    -haproxy-cfg-base-path <path>           # base path to store haproxy config in,                  \
-                                            # will generate a unique directory per run under this,   \
-                                            # defaults to /tmp                                       \
-
-
-    -haproxy <haproxy binary path>          # no required if in your path                            \
-    -dataplane <dataplane-api binary path>  # no required if in your path                            \
-```
\ No newline at end of file
+./haproxy-consul-connect --help
+Usage of ./haproxy-consul-connect:
+  -dataplane string
+    	Dataplane binary path (default "dataplane-api")
+  -enable-intentions
+    	Enable Connect intentions
+  -haproxy string
+    	Haproxy binary path (default "haproxy")
+  -haproxy-cfg-base-path string
+    	Haproxy binary path (default "/tmp")
+  -http-addr string
+    	Consul agent address (default "127.0.0.1:8500")
+  -log-level string
+    	Log level (default "INFO")
+  -sidecar-for string
+    	The consul service id to proxy
+  -sidecar-for-tag string
+    	The consul service id to proxy
+  -stats-addr string
+    	Listen addr for stats server
+  -stats-service-register
+    	Register a consul service for connect stats
+  -token string
+    	Consul ACL token./haproxy-consul-connect --help
+Usage of ./haproxy-consul-connect:
+  -dataplane string
+    	Dataplane binary path (default "dataplane-api")
+  -enable-intentions
+    	Enable Connect intentions
+  -haproxy string
+    	Haproxy binary path (default "haproxy")
+  -haproxy-cfg-base-path string
+    	Haproxy binary path (default "/tmp")
+  -http-addr string
+    	Consul agent address (default "127.0.0.1:8500")
+  -log-level string
+    	Log level (default "INFO")
+  -sidecar-for string
+    	The consul service id to proxy
+  -sidecar-for-tag string
+    	The consul service id to proxy
+  -stats-addr string
+    	Listen addr for stats server
+  -stats-service-register
+    	Register a consul service for connect stats
+  -token string
+    	Consul ACL token
```

Comment:
This is a copy-paste duplicate of the lines above?
```suggestion
```

Reply:
I got lost in my branches... fixed in next commit

Reply:
Not a problem! I will merge as soon as it is fixed!

Reply:
Actually I can commit and merge